app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -67,6 +71,23 @@ export class CreateTag extends React.Component<
             initialValue={this.props.initialName}
             onValueChange={this.updateTagName}
           />
+
+          {this.state.previousTags !== null && (
+            <>
+              <p>Previous Tags</p>
+              {lastThreeTags.length === 0 ? (
+                <>
+                  <Ref>{`No matches found for '${this.state.tagName}'`}</Ref>
```

Comment:
Just noticed in your screenshot (thanks for sharing that) something I overlooked while reviewing it.

I think for this specific scenario, we don't want to use a `Ref`, just normal text 😅 

Sorry about that!